app/views/shared/_tagging.html.erb

```diff
@@ -0,0 +1,104 @@
+                         class: 'js-mainstreams select2',
+                         data: { placeholder: 'Choose mainstream browse pages…' } } %>
+
+          <p class='help-block'>
+            Mainstream browse pages live under <a href="https://www.gov.uk/browse">/browse</a>.
+            Examples: <a href="https://www.gov.uk/browse/working">Working</a> or
+            <a href="https://www.gov.uk/browse/business">Business</a>.
+            They are used to organise mainstream content. Formerly called
+            <i>sections and subsections</i>.
+          </p>
+        </div>
+
+        <div class="form-group" id="edition_parent_input">
+          <%= f.label :parent, "Breadcrumb", class: 'control-label' %>
+          <%= f.select :parent,
+                       Tagging::Linkables.mainstream_browse_pages,
+                       { include_blank: true },
+                       { multiple: false,
+                         class: 'js-primary-mainstream select2',
+                         data: { placeholder: 'Choose a primary mainstream browse page…' } } %>
+
+         <br />
+         <div class="alert mainstream alert-warning hidden">
+           The breadcrumb does not match the browse pages.
+         </div>
+
+
+          <p class='help-block'>
+            If the page has a breadcrumb, it will be this browse page. For example
+            <a href="https://www.gov.uk/benefit-cap-calculator">the breadcrumb on this page</a>.
+            You can only choose one.
+          </p>
+        </div>
+
+        <div class="form-group" id="edition_topics_input">
+          <%= f.label :topics, "Topics", class: 'control-label' %>
+          <%= f.select :topics, Tagging::Linkables.topics,
+                       {},
+                       { multiple: true,
+                         class: 'select2',
+                         data: { placeholder: 'Choose Topics…' } } %>
+
+          <p class='help-block'>
+            Topic pages live under <a href="https://www.gov.uk/topic">/topic</a>.
+            They contain collections of pages on GOV.UK. Example:
+            <a href="https://www.gov.uk/topic/business-tax">Business Tax</a> or
+            <a href="https://www.gov.uk/topic/animal-welfare/pets">Animal Welfare / Pets</a>.
+            Tagging a page with a topic will send out an
+            <a href="https://www.gov.uk/topic/business-tax/vat/email-signup">email alert to subscribers</a>.
+            Formerly called <i>specialist sectors</i>.
+          </p>
+        </div>
+      </div>
+    </div>
+
+    <hr/>
+    <%= f.submit 'Update tags' %>
+<% end %>
+
+
+<script type="text/javascript">
+  $(function () {
+    var $primaryMainstream = $('.js-primary-mainstream');
+    var $mainstreamAlertWarning = $('.mainstream.alert-warning');
+
+    $('.js-primary-mainstream, .js-mainstreams').on('change', function() {
+      $(this).addClass('interacted');
+
+      var mainstreams = $('.js-mainstreams :selected').map( function() {
```

Comment:
That's not the same thing, `.val()` is just returning one value instead of all.
I want all of the selected values so I can check on them.

<img width="1179" alt="screen shot 2016-07-04 at 14 01 13" src="https://cloud.githubusercontent.com/assets/136777/16561206/d215aef0-41ef-11e6-95dc-4596ddbb9f3d.png">